pyfstat/tcw_fstat_map_funcs.py

```diff
@@ -266,6 +266,16 @@ def _optional_imports_pycuda():
     )
 
 
+def _get_transient_fstat_map_features():
+    """Helper function to check available features."""
+    features = {}
+    have_lal = _optional_import("lal")
+    have_lalpulsar = _optional_import("lalpulsar")
+    features["lal"] = have_lal and have_lalpulsar
+    features["pycuda"] = _optional_imports_pycuda()
```

Comment:
`_optional_imports_pycuda` is a conjunction of `bool` values provided by `_optional_import`, hence the return value itself is *always* a `bool` as constructed here.